src/collections/members/sudhanshu-dasgupta/index.mdx

```diff
@@ -0,0 +1,15 @@
+---
+name: Sudhanshu Dasgupta
+position: Contributor
+image_path: ./sudhanshu-dasgupta.jpeg
+github:  sudhanshutech
+twitter: SudhanshuDasgu3
+linkedin: sudhanshu-dasgupta
+location: Madhya Pradesh, India
+bio:As an undergraduate student of Computer Science, I possess a strong passion for web development, as well as extensive knowledge of cloud computing and networking. With a deep interest in open-source technology, I have contributed significantly to various projects, and have found Layer5 to be an exceptionally effective platform for learning.
```

Comment:
@Saksham294 there is syntax error in this file because of that it is not showing my profile in `/members` page

```suggestion
bio: As an undergraduate student of Computer Science, I possess a strong passion for web development, as well as extensive knowledge of cloud computing and networking. With a deep interest in open-source technology, I have contributed significantly to various projects, and have found Layer5 to be an exceptionally effective platform for learning. In addition to my development skills, I have a proven ability to build connections and network with others, which has been demonstrated in my role as a core team lead for a tech club. I have also gained valuable experience mentoring students, further honing my ability to communicate complex ideas in a clear and concise manner. Overall, I am a dedicated and driven individual who is committed to advancing my skills and knowledge in the field of computer science, while contributing to the wider community through open-source technology.
```

Reply:
I'm sorry, didn't know about it. Should I open another issue for this, since this PR is closed?

Reply:
yeah you can, whenever working with yaml it is a good practice to check once in yaml validator